### DISCUSSION TITLE
Can't extend combobox with left click.

### DISCUSSION THREAD

**Discussion by ISDHN** [AvaloniaUI/Avalonia#15644] [Q&A]
I can't extend combobox with left click. There is nothing happened.
However, right click can trigger the combobox  to extend.
Here is the code:

```xml
<Border>
				<RelativePanel>
					<ComboBox RelativePanel.AlignRightWithPanel="True"
							  RelativePanel.AlignVerticalCenterWithPanel="True"
							  Margin="0,0,15,0"
							  Width="120"
							  MaxDropDownHeight="100"
							  IsDropDownOpen="True"
							  Name="Generators">
						<ComboBox.ItemTemplate>
							<DataTemplate>
								<TextBlock Text="{Binding}"/>
							</DataTemplate>
						</ComboBox.ItemTemplate>
					</ComboBox>
				</RelativePanel>
</Border>
```

**Comment by ISDHN** [comment]
Besides, in wsl, I can extend the combobox with left click normally.

**Reply by timunie** [reply]
I don't see anything obvious here. File a minimal sample App please.

**Reply by ISDHN** [reply]
Thank you. Now I found the cause while I try to make the minimal sample App.
```c#
private void Window_PointerPressed(object? sender, Avalonia.Input.PointerPressedEventArgs e)
{
    var pos = e.GetPosition(this);

    var clientRect = this.Bounds.Inflate(-borderWidth);
    if (clientRect.Contains(pos))
    {
        this.BeginMoveDrag(e);  //this line
    }
}
```
But why?

**Reply by timunie** [reply]
check `BeginMoveDrag` code. Most likely it marks the event as handled and thus the ComboBox don't recieve it.

**Reply by ISDHN** [reply]
More likely a problem of ComboBox.  When I pressed on other interactive controls like xxxButton and TabControl and so on the Window_PointerPressed isn't raised. However, press on ComboBox does raise it.

**Reply by ISDHN** [reply]
In Avalonia.Controls, ComboBox.cs:
```c#
        if (_popup?.IsInsidePopup(source) == true)
        {
            e.Handled = true;
            return;
        }
```

It requires  the source of the Pressed event in the DropDown part (_popup).
However, I observe that the soure is either the Border (Backgroud) or the TextBlock (which displays the current selection), and neither of them is inside the popup. So, this event will never be marked as handled.
